v1.1-beta/matrixx-digital-commerce/matrixx-dcp/cnf-testsuite.yml

```diff
@@ -0,0 +1,111 @@
+---
+helm_directory: "matrixx"
+release_name: "cnf"
+helm_install_namespace: "matrixx"
+helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values1685358688.yaml\
```

Comment:
Make this generic so it works in other environments besides your own system @deerowbear